pkg/clients/dynatrace/settings_logmonitoring.go

```diff
@@ -0,0 +1,127 @@
+package dynatrace
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube/logmonitoring"
+	"github.com/Dynatrace/dynatrace-operator/pkg/clients/utils"
+)
+
+type IngestRuleMatchers struct {
+	Attribute string   `json:"attribute,omitempty"`
+	Operator  string   `json:"operator,omitempty"`
+	Values    []string `json:"values,omitempty"`
+}
+
+type logMonSettingsValue struct {
+	ConfigItemTitle string               `json:"config-item-title,omitempty"`
+	Matchers        []IngestRuleMatchers `json:"matchers,omitempty"`
+	Enabled         bool                 `json:"enabled,omitempty"`
+	SendToStorage   bool                 `json:"send-to-storage,omitempty"`
+}
+
+type logMonSettingsItem struct {
+	ObjectID           string              `json:"objectId,omitempty"`
+	LogMonitoringValue logMonSettingsValue `json:"value,omitempty"`
+}
+
+type posLogMonSettingsBody struct {
+	SchemaId      string              `json:"schemaId"`
```

Comment:
done in https://github.com/Dynatrace/dynatrace-operator/pull/4053/commits/13961d08a7523bef40715806129282fe0312e631

I removed every `omitempty` besides scope

a. because an empty scope can be passed
b. we also did it the same in other settings files